### ISSUE TITLE
camelCase doesn't delete original names

### ISSUE THREAD

**Issue by klimashkin** [webpack-contrib/css-loader#368]
`camelCase` option adds new props to style object, not replaces original ones. 

It causes dirt when used with https://github.com/javivelasco/react-css-themr.

For instance, if I have `.back-button {...}` class, css-loader with `camelCase` returns 
```
{
   back-button: '...',
   backButton: '...'
}
```
And if that style object is passed to react-css-themr as `theme={styles} themeNamespace='back'`, it transforms it to 
```
{
   -button: '...',
   button: '...'
}
```
It doesn't make any sense. Is there any reasons why original class name is saved by css-loader?

**Comment by timse** [comment]
this is most likely to keep being backword compatible etc.

why dont you just clean up the object before passing it along?
`styles.filter(style => style.indexOf('-') === -1)`

**Comment by klimashkin** [comment]
@timse In thousand components? : )

**Comment by timse** [comment]
im sure you find a better way :)
but isn't it dangerous anyways to rely on two independent systems to provide correct data for each others api?
you would probably want to have some intermediate step between those two in case one of them changes its api anyways no?

**Comment by michael-ciniawsky** [comment]
Fixed by #420

**Comment by joscha** [comment]
actually, will be fixed by #440 , #420 only removes the duplicate keys (cc @michael-ciniawsky )

**Comment by michael-ciniawsky** [comment]
@joscha Alright 🤣 all my hopes are shattered
